development/roles.md

```diff
@@ -132,7 +132,78 @@ An Apache Hive committer did an excellent write up on their process and much of
 this aligns with our philosophy on maintainers.
 [Read about it](https://cwiki.apache.org/confluence/display/Hive/BecomingACommitter).
 
-[You can view all maintainers on GitHub.](https://github.com/orgs/trinodb/teams/maintainers/members)
+The following community members are Trino maintainers:
+
+<table style="width: 30%">
+  <thead>
+    <tr>
+      <th><i class="fab fa-github"></i></th>
+      <th>Full name</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="https://github.com/arhimondr">arhimondr</a></td>
+      <td>Andrii Rosa</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/dain">dain</a></td>
+      <td>Dain Sundstrom</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/ebyhr">ebyhr</a></td>
+      <td>Yuya Ebihara</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/electrum">electrum</a></td>
+      <td>David Phillips</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/findepi">findepi</a></td>
+      <td>Piotr Findeisen</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/hashhar">hashhar</a></td>
+      <td>Ashhar Hasan</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/kasiafi">kasiafi</a></td>
+      <td>Kasia Findeisen</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/kokosing">kokosing</a></td>
+      <td>Grzegorz Kokosiński</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/losipiuk">losipiuk</a></td>
+      <td>Łukasz Osipiuk</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/martint">martint</a></td>
+      <td>Martin Traverso</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/phd3">phd3</a></td>
+      <td>Pratham Desai</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/Praveen2112">Praveen2112</a></td>
+      <td>Praveen Krishna</td>
+    </tr>
+    <tr>
+      <td><a href="https://github.com/raghavsethi">raghavsethi</a></td>
```

Comment:
@martint @dain @electrum do we still want to include Raghav on the list?

Reply:
Well.. he is on the github list .. so unless we change his status and so on there as well we should leave for now. All that is basically a separate topic.

Reply:
Yeah, the change is otherwise good to go, just suggesting we make both changes so that people don't reach out to him thinking he's still involved.